src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
glassez, is right. qt4 uses toAscii() to convert and qt5 uses toUtf8().
I suppose you code using qt5 and your intention is to use utf8. So in this case you should use `Utils::String::toStdString()`.
Btw, did you test with utf8 + windows + non-latin path? Did it work?